app/src/test/kotlin/com/intive/picover/profile/viewmodel/ProfileViewModelTest.kt

```diff
@@ -34,7 +34,12 @@ class ProfileViewModelTest : ShouldSpec(
 		val uri: Uri = mockk()
 		val authRepository: AuthRepository = mockk(relaxed = true)
 		val toastPublisher: ToastPublisher = mockk(relaxed = true)
-		val textValidator: TextValidator = mockk()
+		// TODO: Implement separate TextValidator based on screen purpose
+		val textValidator: TextValidator = TextValidator.build {
```

Comment:
why you use real object here instead of mock?

Reply:
Mocking every single response for TextValidator it's pointless. In this case for now it's better to use just a real object, but it shows that there is a problem with testing TextValidator in ViewModel so it should be refactored in the next step.